CalibCalorimetry/EcalTrivialCondModules/interface/EcalLaserCondTools.h

```diff
@@ -0,0 +1,93 @@
+#ifndef SRCONDACCESS_H
+#define SRCONDACCESS_H
```

Comment:
```suggestion
#define EcalLaserCondTools_h
```

This name conflicts with [`SimCalorimetry/EcalSelectiveReadoutProducers/interface/EcalSRCondTools.h`](https://github.com/cms-sw/cmssw/blob/master/SimCalorimetry/EcalSelectiveReadoutProducers/interface/EcalSRCondTools.h#L1).

Reply:
@grasph Please fix also
`#ifndef` at L.1 and the comment at the last line `#endif // EcalLaserCondTools_h not defined`